modules/nw-cfg-tuning-interface-cni.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-operator.adoc
+:_content-type: PROCEDURE
+[id="nw-configuring-tuning-cni_{context}"]
+= Configuring the tuning CNI
+
+The following procedure shows an example of configuring the tuning CNI to change the interface level network `net.ipv4.conf.IFNAME.accept_redirects` sysctl. This example enables the acceptance and sending of ICMP redirected packets.
+
+.Procedure
+
+. Create a network attachment definition, such as `tuning-example.yaml`, with the following content:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
```

Comment:
When I tried to apply this yaml I've got:
oc create -f /tmp/test_nad.yaml
`Error from server: error when creating "/tmp/test_nad.yaml": admission webhook "multus-validating-config.k8s.io" denied the request: configuration string is not in JSON format`
The format of json is invalid. Check my comments below

Reply:
Example of valid nad.

```
apiVersion: "k8s.cni.cncf.io/v1"
kind: NetworkAttachmentDefinition
metadata:
  name: tuningnad
  namespace: default
spec:
  config: '{
    "cniVersion": "0.4.0",
    "name": "tuningnad",
    "plugins": [{
	"type": "bridge"
      },
      {
	"type": "tuning",
	"sysctl": {
	   "net.ipv4.conf.IFNAME.accept_redirects": "1"
        }
      }
  ]
}
```

Reply:
missing a }'. Fixed and applied to my cluster - thanks